cmd/zinject/translate.c

```diff
@@ -120,7 +120,7 @@ parse_pathname(const char *inpath, char *dataset, char *relpath,
 #else
 	if ((fp = fopen(MNTTAB, "r")) == NULL) {
 #endif
-		(void) fprintf(stderr, "cannot open /etc/mtab\n");
+		(void) fprintf(stderr, "cannot open /proc/self/mounts\n");
```

Comment:
And this minor tweak, https://github.com/rlaager/zfs/commit/cd3fd0948888e91305985d95a6640c8f1b964685